locals.tf

```
@@ -177,12 +177,12 @@ locals {
             enabled = true
           }
         },
-        var.cluster_issuer == "ca-issuer" ? {
+        var.cluster_issuer == "ca-issuer" || var.cluster_issuer == "letsencrypt-staging" ? {
           volumeMounts = [
             {
               name      = "certificate"
-              mountPath = "/etc/ssl/certs/ca.crt"
-              subPath   = "ca.crt"
+              mountPath = format("/etc/ssl/certs/%s", var.ssl_secret_key)
+              subPath   = var.ssl_secret_key
```

```suggestion
              mountPath = format("/etc/ssl/certs/%s", var.cluster_issuer == "letsencrypt-staging" ? "tls.crt" : "ca.crt")
              subPath   = var.cluster_issuer == "letsencrypt-staging" ? "tls.crt" : "ca.crt"
```

@hbollon So, from what I've gathered, when there is no self-signed Certificate Authority, cert-manager won't add the `ca.crt` to the Kubernetes secret, as it does not exist. That is the case with `letsencrypt-staging`. In that case, the secret will only be the SSL certificate itself and contain only the `tls.crt` and `tls.key` fields and I'm not aware that these fields can be anything else than this, so I do not see the need to variabilize this field and I think my suggestion would be more appropriate.

Does anyone on @camptocamp/is-devops-stack think we have a reason to keep this variable?

---

Indeed, it seems more appropriate this way :wink: 
I've updated the PR accordingly (and tested on SKS btw)